README.md

```diff
@@ -67,7 +65,7 @@
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-- [Live Demo Link](https://rhaegar121.github.io/To-Do-list/dist/)
+- [Live Demo Link]()
```

Comment:
- [ ] Live demo link in your ReadMe is broken, kindly endeavor to fix or remove it if this app is not yet deployed.